nautobot_ssot/migrations/0003_auto_20220927_1352.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.15 on 2022-09-27 13:52
```

Comment:
Can you rename this migration to something meaningful? Otherwise this looks good!

Reply:
Yes, there you go!